src/dags/rioolnetwerk.py

```diff
@@ -4,13 +4,10 @@
 from airflow import DAG
 from airflow.providers.postgres.operators.postgres import PostgresOperator
 from common import (
-    OTAP_ENVIRONMENT,
     DATASTORE_TYPE,
-    SLACK_ICON_START,
     MessageOperator,
```

Comment:
No SHARED_DIR needed?

Reply:
It uses underwater a temporary named folder for it's temporary data storage. It's included in the SwiftLoadSqlOperator. That operator is doing the download, unzip and ingestion in one go